bin/generate-docs.ts

```diff
@@ -0,0 +1,147 @@
+//
```

Comment:
This is really cool, but seems largely untested. Could we add some tests to cover this even if at a high-level?

Reply:
It's coming from here and didn't find test: https://github.com/reside-eng/verify-git-tag-action/blob/main/bin/generate-docs.ts
But you're definitely right it should be tested
I'm not sure to be the best one to implement tests for this but I can give it a try